src/javascripts/ng-admin/Main/config/translate.js

```diff
@@ -38,6 +38,7 @@ export default function translate($translateProvider) {
       'NEXT': 'Next »',
       'DETAIL': 'Detail',
       'STATE_CHANGE_ERROR': 'State change error: {{ message }}',
+      'STATE_FORBIDDEN_ERROR': 'A server 403 error occured: {{ message }}',
```

Comment:
Let use current message, as it will be useful for users to report problem to their developper. Moreover, it is easily overridable now.